src/commands/dev.ts

```diff
@@ -290,19 +285,32 @@ export default class Dev extends BaseCommand {
     const docker_compose_runnable = DockerComposeUtils.dockerCompose(compose_args, { stdout: 'pipe', stdin: "ignore" });
 
     let is_exiting = false;
+    let halt_output = false;
     this.setupSigInt(() => {
       if (is_exiting) {
         return;
       }
       is_exiting = true;
-      docker_compose_runnable.kill('SIGTERM');
+
+      if (process.platform === "win32") {
+        // On windows, there's no clean way to send a SIGTERM/SIGINT equivalent because interrupts don't exist.
+        // Instead, we run `docker compose stop` and stop logging output from the containers.
+        halt_output = true;
+        DockerComposeUtils.dockerCompose(['-f', compose_file, '-p', project_name, 'stop']);
```

Comment:
Should we just run this for all platforms? Let docker handle the unique platform constraints.

Reply:
Yeah probably - going to test doing this on all platforms and make sure it behaves the same way